keyboards/evyd13/atom47/keymaps/phsc138/keymap.c

```diff
@@ -0,0 +1,390 @@
+/* Copyright 2022 PHSC138
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+// These are all aliases for the function layers.
+enum custom_layers {
+    _BASE,
+    _PROG,
+    _GAME,
+    _FN,
+    _FN1,
+    _NUM
+};
+
+
+// Tap Dance
+typedef struct {
+    char is_press_action;
+    int state;
+} tap;
+
+enum {
+    SINGLE_TAP = 1,
+    SINGLE_HOLD = 2,
+    DOUBLE_TAP = 3,
+    DOUBLE_HOLD = 4,
+    DOUBLE_SINGLE_TAP = 5, // Send two single taps
+    TRIPLE_TAP = 6,
+    TRIPLE_HOLD = 7
+};
+
+// Tap dance enums
+enum {
+    PN_SWAP = 0,
+    LAPO = 1,
+    LCPO = 2,
+    RAPC = 3,
+    RCPC = 4,
+    D20 = 5
+};
+
+enum custom_keycodes {
+    PROF_MAC = SAFE_RANGE
```

Comment:
```suggestion
    PROF_MAC = QK_USER,
```

Reply:
This seems trivial?
`#define SAFE_RANGE             QK_USER`

Reply:
`SAFE_RANGE` will eventually be removed, that's why that define is there.

Reply:
Thanks for the explanation